ci-operator/config/open-cluster-management/kube-state-metrics/OWNERS

```diff
@@ -0,0 +1,11 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- morvencao
```

Comment:
Highly recommend you have more than one approver. If you're not available, there will be no way to get updates made to the Prow config.

Reply:
Done.